static/scss/search-page.scss

```diff
@@ -365,3 +365,20 @@
     color: rgba(0,0,0,0.8);
   }
 }
+
+.sk-hierarchical-menu-list {
+  width: 100%;
+
+  .sk-hierarchical-menu-list__root {
+    margin-left: 10px;
+    margin-right: 10px !important;
```

Comment:
Can you add a comment explaining the `!important` ? 

@singingwolfboy is encouraging us to add a comment whenever we use !important so that the next developer who looks at this will know what we're overriding.